#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_ignore_errors_from_fstype.dpatch by Michael Prokop <mika@grml.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Ignore errors from fstype
##
## Redirect stderr to /dev/null, otherwise we'll get
## "stdin: error 0" for some partitions during bootup.
## 

@DPATCH@

diff -r 6b72cefbc61b scripts/live-helpers
--- a/scripts/live-helpers	Mon Sep 15 17:32:23 2008 +0200
+++ b/scripts/live-helpers	Mon Sep 15 17:34:17 2008 +0200
@@ -77,7 +77,7 @@
 	    return
 	fi
 
-	eval $(fstype < ${1})
+	eval $(fstype < ${1} 2>/dev/null)
 
 	if [ "${FSTYPE}" != "unknown" ]
 	then
